docs/adr/0015-copy-source-to-files.md

```diff
@@ -0,0 +1,25 @@
+# *Copying files from the source to the files folder*
+
```

Comment:
Please add a decision drivers:

- support for scripting languages
- support for compiled languages